test/multi_tenancy.sh

```diff
@@ -74,51 +80,60 @@ function create_user_and_assign_to_projects() {
 }
 
 function add_message_to_index() {
-    # project is $1
-    # message is $2
-    # espod is $3
+
+    local namespace=$1
     local project_uuid=$( oc get project $1 -o jsonpath='{ .metadata.uid }' )
     local index="project.$1.$project_uuid.$(date -u +'%Y.%m.%d')"
```

Comment:
Since you are making this change in context of the new data model i would go ahead and structure the setup accordingly instead of using the legacy format:
* data lives in an index like (recalling from memory): `logs.apps-000001`
* index is aliased like `logs.apps`

Or maybe it doesnt matter since we are relying on the alias and OpenDistro will secure the actual index.  Eitherway the index name likely does not need to be as complicated now as it was previously

Reply:
simplified the index name